ccmlib/scylla_node.py

```diff
@@ -1785,6 +1785,78 @@ def wait_for_compactions(self,
                                      column_family=column_family,
                                      idle_timeout=idle_timeout)
 
+    def run_scylla_types(self, action: str, values: List[str], types: List[str], additional_args: Optional[List[str]]=None, text=True):
```

Comment:
We can use a convenience type to help people with the `types`:
```python

class TypeKind(Enum):
    Simple = 1
    ClusteringKey = 2
    PartitionKey = 3

class scylla_types:
    kind: TypeKind
    types: Tuple[str]

    def __init__(self, kind, types):
        self.kind = kind
        self.types = types

    @staticmethod
    def make_simple(type: str):
        return Type(TypeKind.Simple, (type,))

    @staticmethod
    def make_clustering_key(*types):
        return Type(TypeKind.ClusteringKey, types)

    @staticmethod
    def make_partition_key(*types):
        return Type(TypeKind.PartitionKey, types)
```

Maybe we can add an num with all the different types too, to save people a lookup.
See https://github.com/scylladb/scylladb/blob/master/docs/dev/cql3-type-mapping.md for a table of all the types.

Reply:
I would not add the enum with all the different type, this PR would explode with content. I would like to keep this simple. I like the idea of ScyllaType class with kind, only I would use class_method instead. I think this class could solve `values `and `list` handling

Reply:
Yes, I later realized that adding an enum with all types is problematic also because some types are compound, e.g. lists and that is hard to express with an enum. So we can leave type as a free string, but let's add a link to the above document in the comments so people can quickly find the proper type names.

Reply:
I have updated the code based on your feedback. Let me know your comments on that.